modules/script/script-impl/index.d.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference path="global.d.ts" />
+
+declare const app: {
+    name: string;
+    version: string;
+    config: Record<string, string>;
```

Comment:
I have been thinking if `config` should just be `Record<string, string | undefined>`, to force the developer to handle the `undefined` case, because we are making a pretty big assumption that a given config always exists.